src/main/webapp/components/template/slider/slider.html

```diff
@@ -0,0 +1,3 @@
+<div id="slider"></div>
```

Comment:
You should/must not use slider as id here: this is a reusable component, and id should not repeat. In reusable directive (reusable multiple times per page), you should not use any ids.